presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -121,6 +122,33 @@ public void testCreateTableAs()
         execute("DROP TABLE table_all_types_copy");
     }
 
+    @Test
```

Comment:
Add test cases for the table with multiple partitions key and multiple clustering keys